### PR TITLE
chore: state: add cli to get the manifest & actor cid for network v16 & up

### LINKED ISSUE #8661: Util to get the actor <> CID map
### Checklist

- [X] This is **not** a new feature or an enhancement to the Filecoin protocol. If it is,  please open an [FIP issue](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0001.md).
- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

users of lotus should be able to easily get the actor CIDs they are running for validation checking and so on.

### Describe the solution you'd like

something like `lotus state actor-cids`

and output

Actor       CID
system    bayxxxx
account  bayxxxxxxx

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jennijuju**
## Related Issues

resolves #8661 

image
image
image

## Additional Info

image

## Checklist

Before you mark the PR ready for review, please make sure that:
- [x] All commits have a clear commit message.
- [x] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [ ] This PR has tests for new functionality or change in behaviour
- [ ] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] CI is green

### CODE REVIEW

File: cli/state.go [hunk 1]
```diff
@@ -1872,3 +1878,82 @@ var StateNtwkVersionCmd = &cli.Command{
...
+		if !ok {
+			return xerrors.Errorf("cannot get manifest CID")
+		}
+		fmt.Printf("Manifest CID: %v\n", manifestCid)
+
+		tw := tabwriter.NewWriter(os.Stdout, 2, 4, 2, ' ', 0)
+		_, _ = fmt.Fprintln(tw, "\nActor\tCID\t")
+
+		for _, name := range []string{
+			actors.AccountKey,
```
Comment (arajasek) [inline cli/state.go:1937]: Can I bug you to make one refactor? Add:

```
func GetBuiltinActorsKeys() []string {
	return []string{
		AccountKey,
		CronKey,
		InitKey,
		MarketKey,
		MinerKey,
		MultisigKey,
		PaychKey,
		PowerKey,
		RewardKey,
		SystemKey,
		VerifregKey,
	}
}
```

to chain/actors/manifest.go, and use that here (and in chain/actors/manifest.go).

**Comment by jennijuju** [comment]
image

File: chain/actors/manifest.go [hunk 2]
```diff
@@ -140,3 +129,19 @@ func CanonicalName(name string) string {
 
 	return name
 }
+
+func GetBuiltinActorsKeys() []string {
```
Comment (arajasek) [inline chain/actors/manifest.go:133]: Let's put this at the top of the file (right after the keys themselves are defined).

**Review by arajasek — APPROVED** [review]
with a nit!

**Pull request merged**